### DISCUSSION TITLE
Folder structure without year

### DISCUSSION THREAD

**Discussion by dosmehobbit** [pablo33/Shotwell-event2folder#31] [Ideas]
Hi, nice script! It works out fine.

I use it to create a "Les Pas" gallery for my Nextcloud foto gallery.

For this I would need the directories of the events in one main folder instead of seperated by years.

Is this possible to add as an configuration?

Example:

/2020/EventA
/2020/EventB
/2021/EventC

will become:

/EventA
/EventB
/EventC

**Comment by pablo33** [comment]
Hi!, Nice to see someone else who use Shotwell and runs this code.

Yes, i think it could be possible. Let me see if i can implement it as an option. Maybe a  _flat tree_ option: True / False (default).


Regards.

**Comment by pablo33** [comment]
Coded as a new feature. Just save and run the code to add the new option to your config file.

**Reply by dosmehobbit** [reply]
I'll try out later. But thanks for your fast reply. Thx. ;-)
